### DISCUSSION TITLE
Have table support plan？

### DISCUSSION THREAD

**Discussion by qqke** [emilk/egui#133] [Ideas]
Have table support plan？

**Comment by emilk** [comment]
The new [`Grid` layout](https://docs.rs/egui/0.8.0/egui/struct.Grid.html) can already be used as a table. I mean to add more features to it as needed (e.g. resizing column widths with the mouse)

**Reply by qqke** [reply]
Looking forward to it

**Reply by g-braeunlich** [reply]
If you are interested: here is an example on how to auto generate tables: Some ideas #272

**Reply by emilk** [reply]
There is now an issue for it: https://github.com/emilk/egui/issues/296
